src/main/java/com/eliasnogueira/driver/TargetFactory.java

```diff
@@ -74,16 +77,23 @@ private RemoteWebDriver createRemoteInstance(MutableCapabilities capability) {
 
     private RemoteWebDriver createTestContainersInstance(MutableCapabilities capabilities) {
         String browser = capabilities.getBrowserName();
+        BrowserWebDriverContainer<?> driverContainer = null ;
+        if (browser.equalsIgnoreCase( CHROME.toString())) {
```

Comment:
I believe this could be part of the `BrowserFactory` class as you now have added the ability to use testcontainers for different browsers, so we would have a single source of truth.

You can follow the same approach applied to the local or remote driver.